File: checkinpromobile/lib/src/model/NativeRequest.dart
import 'package:json_annotation/json_annotation.dart';

part 'NativeRequest.g.dart';

@JsonSerializable()
class NativeRequest {
  @JsonKey(name: 'actions')
  String actions;

  @JsonKey(name: 'data')
  Map<String, dynamic> data;

  NativeRequest(this.actions, this.data);

  NativeRequest._();

  factory NativeRequest.fromJson(Map<String, dynamic> json) => _$NativeRequestFromJson(json);

  Map<String, dynamic> toJson() => _$NativeRequestToJson(this);
}

File: checkinpromobile/android/app/src/main/kotlin/vn/com/unit/checkinpromobile/NativeRequest.kt
package vn.com.unit.checkinpromobile

class NativeRequest {
    var actions: String? = null

    var data: PrinterInfor? = null

    constructor()

    constructor(actions: String?, data: PrinterInfor?) {
        this.actions = actions
        this.data = data
    }
}